GitUp/Application/AppDelegate.m

```diff
@@ -466,7 +462,13 @@ static CFDataRef _MessagePortCallBack(CFMessagePortRef local, SInt32 msgid, CFDa
   XLOG_DEBUG_CHECK(input);
   NSDictionary* output = [(__bridge AppDelegate*)info _processToolCommand:input];
   XLOG_DEBUG_CHECK(output);
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wunguarded-availability"
+  // The deprecation for this method in the macOS 10.14 SDK marks the incorrect
+  // version for its introduction. However, it is useful to keep availability
+  // guards on in general. FB6233110
   return CFBridgingRetain([NSKeyedArchiver archivedDataWithRootObject:output]);
+#pragma clang diagnostic pop
```

Comment:
So the secure encoding API isn't available in 10.13? Is that what you're saying here?

Reply:
No, I am not saying that. `+archivedDataWithRootObject:` has been around since 10.0. When it was deprecated in the 10.14 SDK, it was marked as `API_DEPRECATED(…, macosx(10.11, 10.14), …)`. That says the method was introduced in 10.11, which it was not. Using the newer `+archivedDataWithRootObject:requiringSecureCoding:error:` would not address the warning it adds, it's hitting on availability _in the other direction_.